docs/summits/2025_03/call_to_action.md

```diff
@@ -1 +1,32 @@
 # We need your help !
+
+Thank you for attending this WheelNext Summit (or reading its website materials)!
+Your participation in and contributions during and after this event matter -
+we're collectively trying to achieve some ambitious goals, and we'll need all
+the help we can get.
+
+A first goal is community building - WheelNext is young, and we're rapidly trying
+to pull together as many stakeholders as possible: companies, open source projects,
+and individual Python packaging contributors. Meeting others with similar goals
+and pain points helps collaborate in the future, and building an understanding
+of pain points different from our own will lead to better discussions and
+solutions.
+
+Concretely, here are ways to contribute. During the event:
+
+- Share your own goals, pain points, and ideas
+- Pick one of the 3 workshops to attend and collaborate on a small concrete deliverable
+- Find out how you can help accelerate WheelNext development (prototypes, PEPs,
+  working out use cases & user stories, etc.)
+
+After the event:
+
+- Share news about WheelNext with others who might find it interesting or may want to pitch in
+- Help review or improve PEP drafts, website materials, and other content
+- Test current prototypes on your own use cases and provide feedback or fixes
+- If you're a maintainer of a packaging-related tool: please think about how
+  the designs being developed in WheelNext will be implemented in or affect
+  your tool, and ensure you're happy with the design and impact of it!
+- If you're a package author affected by limitations in Python packaging that
+  seem in scope for WheelNext: please ensure that your use case is captured and
+  addressed.
```

Comment:
We need to add something along the lines of "one of the most important contribution one can make is public support in the open source world - in their name, their project name or their company company - name on DPO.

We struggled on our previous proposals because we didn't have "the community support", it seemed lacking consensus and the conversation rapidly stalled.

I would really add almost a separate paragraph just on that. **If you only can do one thing, please do that**

Reply:
Good point, added.

That was exactly my thought about `.rim` files - I quite liked the idea, but lacked the time to weigh in at that point. The lack of discussion was odd.

Reply:
It took us a long time to find a recipe that would allow people to join. I hope now we have a working formula that will greatly help to address this issue.

I have no doubt we have the right people to address any technical challenge. The community aspect is the tough part